### ISSUE TITLE
`color-function-notation` show error for scss `rgba` function

### ISSUE THREAD

**Issue by qmhc** [stylelint-scss/stylelint-scss#563]
Will take the `color-function-notation` error when using scss rgba function like:
```scss
.a {
  color: rgba($base-color, 0.8);
}
```

A config `'color-function-notation': 'modern'` extends from `stylelint-config-standard` (using `stylelint-config-standard-scss`), it cannot distinguish rgba between css and scss.

**Comment by kristerkari** [comment]
Thanks @qmhc ! 

This is actually not the correct project for that rule.

Could you please open this issue in the `stylient-config-standard-scss` repo instead? https://github.com/stylelint-scss/stylelint-config-standard-scss

We can move the discussion there. 👍

**Comment by qmhc** [comment]
Thanks @kristerkari , please help to move this issue to there.

**Comment by rgant** [comment]
This can likely be closed now.
